google/cloud/storage/internal/retry_client.cc

```diff
@@ -154,6 +154,42 @@ bool UploadChunkOnFailure(RetryPolicy& retry_policy, int& count,
   return retry_policy.OnFailure(status);
 }
 
+Status RetryError(Status const& last_status, RetryPolicy const& retry_policy,
+                  char const* error_message) {
```

Comment:
`error_message` seems a little misleading in the context of `"... in <error_message>"`.

Previously we seem to have used `char const* location` or `char const* function_name`.

Reply:
Fixed